### PR TITLE
Build dev doc

### PR DESCRIPTION
**Pull request by LysandreJik**

### CODE REVIEW

**Comment by LysandreJik** [comment]
Will only need to change the target of the `git add` directive and it should be good to go

**Comment by sgugger** [comment]
As said in private, this won't work for branches that have a different origin than `transformers`, since those won't get access to the secrets for security reasons.

File: .github/workflows/build_dev_documentation.yml [hunk 1]
```diff
@@ -74,29 +70,19 @@ jobs:
...
           cd ..
+          WRITE=`cat /mnt/WRITE`
+          
+          rm -rf doc-build-dev
+          git clone https://HuggingFaceDocBuilder:$WRITE@github.com/huggingface/doc-build-dev
 
       - name: Make documentation
         run: |
-          doc-builder build transformers transformers/docs/source --build_dir doc-builder/build --notebook_dir notebooks/transformers_doc --clean
+          doc-builder build transformers transformers/docs/source --build_dir doc-build-dev/$PR_NUMBER --notebook_dir notebooks/transformers_doc --clean
```
Comment (sgugger) [inline .github/workflows/build_dev_documentation.yml:80]:
```suggestion
          doc-builder build transformers transformers/docs/source --build_dir doc-build-dev --notebook_dir notebooks/transformers_doc --clean --version pr_$PR_NUMBER
```

File: .github/workflows/delete_dev_documentation.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: Setup git
+        run: |
+          git config --global user.name "Hugging Face"
+          git config --global user.email transformers@huggingface.co
+          
+      - name: Push to repositories
+        run: |
+          cd doc-build-dev
+          rm -rf $PR_NUMBER
```
Comment (sgugger) [inline .github/workflows/delete_dev_documentation.yml:32]:
```suggestion
          rm -rf transformers/$PR_NUMBER
```

**Pull request closed without merging**